src/content/get-started/install/openshift.mdx

```diff
@@ -0,0 +1,192 @@
+```
+
+### Copy OpenShift wildcard certificate to the `okteto` namespace
+
+OpenShift provides the wildcard certificate for the wildcard domain `apps.<your-openshift-subdomain>` . We recommend reusing it when installing Okteto.
+
+To do that, copy the certificate to the `okteto` namespace (replace `<cluster-name>` with your cluster name):
+
+```yaml
+kubectl get secret <clustername>-primary-cert-bundle-secret -n openshift-ingress -o yaml | sed 's/namespace: openshift-ingress/namespace: okteto/' | kubectl apply -f -
+```
+
+### Get the OpenShift Ingress Controller IP
+
+We recommend installing Okteto behind the Openshift Ingress controller. To do this, remember the IP of your Openshift Ingress Controller:
+
+```bash
+kubectl get svc router-default -n openshift-ingress -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+```
+
+We will refer to this value using `<ingress-controller-ip>`.
+
+:::warning
+Installing Okteto behing the OpenShift Ingress controller means the following features aren't available:
+- [Auto wake on incoming requests](admin/cleanup.mdx#inactive-applications-and-resources)
+- [Private Endpoints](core/endpoints/private-endpoints.mdx)
+:::
+
+## Installing Okteto
+
+Okteto is installed using a Helm chart. Let's start the process:
+
+### Add the Okteto Helm repository
+
+You'll need to add the Okteto Helm repository to be able to install Okteto:
+
+```bash
+helm repo add okteto https://charts.okteto.com
+helm repo update
+```
+
+### Create the Helm configuration file
+
+In order to install Okteto you need to first create a `config.yaml` for the installation process.
+
+Create the following Helm configuration file for Okteto in, for example, a file called `config.yaml`.
+Remember to replace `<<your-okteto-license>>`, `<your-openshift-subdomain>`, `<cluster-name>` and `<ingress-controller-ip>` with your own values:
+
+```yaml
+license: "REPLACE ME WITH YOUR OKTETO LICENSE"
+
+openshift: # supports Openshift Routes as Okteto Endpoints
+  enabled: true
+
+subdomain: apps.<your-openshift-subdomain>
+
+wildcardCertificate: # reuse your OpenShift wildcard certificate
+  create: false
+  name: <cluster-name>-primary-cert-bundle-secret
+
+ingress-nginx:
+  enabled: false
+
+okteto-nginx:
+  enabled: false
+
+ingress:
+  oktetoIngressClass: openshift-default
+  class: openshift-default
+  forceIngressClass: true
+  ip: <ingress-controller-ip>
+
+cluster:
+  endpoint: https://api.<your-openshift-subdomain>:6443
+    
+user:
+  extraRoleBindings:
+    enabled: true
+    roleBindings:
+      okteto:
+        - system:openshift:scc:anyuid
+```
+
+:::tip
+_Note: This is the minimum configuration. Check our [Helm configuration](self-hosted/helm-configuration.mdx) docs to learn more_
+:::
+
+### Installing the Okteto Helm chart
+
+Install the latest version of Okteto by running:
+
+<CodeBlock language="bash">
+  {`helm upgrade --install okteto okteto/okteto -f config.yaml --namespace=okteto --create-namespace --version=${variables.chartVersion}`}
```

Comment:
do we still need the`--create-namepace` param if we told them the prerequisites to create the namespace?

Reply:
removed